packages/catalog-search/src/LearningTypeRadioFacet.jsx

```diff
@@ -115,25 +115,25 @@ const LearningTypeRadioFacet = ({ enablePathways }) => {
               </Dropdown.Item>
             )
           }
-          {features.ENABLE_VIDEO_CATALOG
-            && (
-              <Dropdown.Item as="label" className="mb-0 py-3 d-flex align-items-center">
-                <Input
-                  type="radio"
-                  checked={typeVideoSelected}
-                  className="facet-item position-relative mr-2 mb-2"
-                  onChange={() => handleInputOnChange(LEARNING_TYPE_VIDEO)}
-                  data-testid="learning-type-videos"
-                />
-                <span className={classNames('facet-item-label', { 'is-refined': typeVideoSelected })}>
-                  <FormattedMessage
-                    id="search.facetFilters.learningType.videos"
-                    defaultMessage="Videos"
-                    description="Title for the learning type facet filter to return videos only"
-                  />
-                </span>
-              </Dropdown.Item>
-            )}
+          {features.ENABLE_VIDEO_CATALOG && enableVideos
```

Comment:
[nit / food for thought] With the ability to conditionally include videos here based on `enableVideos`, is the `features.ENABLE_VIDEO_CATALOG` check needed, given the consuming MFE could abstract the feature flag check into the `enableVideos` prop?

Example:

```
enableVideos={features.ENABLE_VIDEO_CATALOG && hasActivatedAndCurrentSubscription}
```